### PR TITLE
Add find me button for location enabled

### LINKED ISSUE #29: setMyLocationEnabled(true) should also show find me button
If we want to model the prevailing map behavior then showing a UI component to center on the users current location (and start tracking updates) will also be enabled when `setMyLocationEnabled(boolean)` is set to true.

https://developers.google.com/android/reference/com/google/android/gms/maps/GoogleMap.html#setMyLocationEnabled(boolean)

### PR DESCRIPTION
**Pull request by sarahlensing**
Closes #29

### CODE REVIEW

File: library/src/main/java/com/mapzen/android/TangramMap.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package com.mapzen.android;
+
+import android.content.Context;
+import android.util.AttributeSet;
+
+/**
+ * Entry point for interation with Tangram map. Wrapped inside of {@link MapView}.
```
Comment (ecgreb) [inline library/src/main/java/com/mapzen/android/TangramMap.java]: interaction + integration = interation ?

;)

File: library/src/main/res/layout/view_map.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (ecgreb) [inline library/src/main/res/layout/view_map.xml]: Can this be a `<merge>` tag to reduce layout depth?

Right now `MapView` extends `FrameLayout` so we have a `RelativeLayout` inside a `FrameLayout`. Perhaps `MapView` should instead extend `RelativeLayout` and then we use a `<merge>` tag here to wrap the map and find me button?

File: library/src/main/java/com/mapzen/android/TangramMap.java (hunk 1 above)
Comment (sarahlensing) [inline library/src/main/java/com/mapzen/android/TangramMap.java]: :) ...updated to interaction!

File: library/src/main/res/layout/view_map.xml (hunk 2 above)
Comment (sarahlensing) [inline library/src/main/res/layout/view_map.xml]: Indeed! Nice catch, updated.

**Pull request merged**